src/open_producten/producten/migrations/0007_remove_product_bsn_remove_product_kvk_eigenaar.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 4.2.17 on 2025-03-14 13:08
```

Comment:
Not super important, but sometimes it can be good to rename the migration to be a bit more clear as to what it does (e.g. here I would probably call it something like  `0007_add_product_eigenaar`)